how-to/customize-home-templates/client/src/home.ts

```diff
@@ -32,7 +32,7 @@ export async function register() {
 		response: HomeSearchListenerResponse
 	): Promise<HomeSearchResponse> => {
 		try {
-			const query = request.query.toLowerCase();
+			const queryLower = request.query.toLowerCase();
```

Comment:
Are we only using queryLower to check for ? if so to we need to lower it at all?

Reply:
I kept it like this for consitency with all the other sample, becuase apps are usually checked case insentive, and the full case version is passed to integrations.